.github/workflows/notify-mm-blog.yml

```diff
@@ -0,0 +1,32 @@
+name: notify-mm-blog
+on:
+  push:
```

Comment:
Don't we want to limit this to `master`? Otherwise it will kick in on new files on branches, right?

```suggestion
  push:
    branches: 
      - master
```

Reply:
no, we want only in that path

Reply:
Can't we do both? path and branch?

Reply:
why?

Reply:
Correct me if I'm wrong, but doesn't the current code post a message if we add a new file to `site/content/blog/*.md`? I think we only want to post it, when the file gets added to `master`, not when it get added to the feature branch.

Reply:
sounds fair